src/main/java/ch/njol/skript/expressions/ExprRawName.java

```diff
@@ -56,64 +49,44 @@
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.PropertyExpression;
-import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.skript.lang.util.SimpleExpression;
-import ch.njol.skript.util.PotionEffectUtils;
 import ch.njol.util.Kleenean;
 
 @Name("Raw Name")
 @Description("Raw Minecraft material name for given item. Note that this is not guaranteed to give same results on all servers.")
 @Examples("raw name of tool of player")
-@Since("unknown (2.2)")
-public class ExprRawName extends SimpleExpression<String> {
+@Since("unknown (2.2-Fixes-v10), 2.2-dev35")
```

Comment:
It would be: 2.2-Fixes-v10, 2.2-dev35 (what you added in this version)

Reply:
All that was changed was it converting over to a SimplePropertyExpression @Blueyescat and some others don't agree with just putting that. I originally had (converted to SimplePropertyExpression)

Reply:
![image](https://user-images.githubusercontent.com/19474570/37340833-6b1356b8-26d0-11e8-99e8-5e207b612352.png)